src/main/thrift/parquet.thrift

```diff
@@ -545,12 +545,15 @@ struct ColumnMetaData {
 }
 
 struct ColumnChunk {
-  /** File where column data is stored.  If not set, assumed to be same file as
-    * metadata.  This path is relative to the current file.
-    **/
+  /** Deprecated and unused. Please ignore. **/
```

Comment:
@julienledem We discussed that this field is unused, but I'm not so sure any more. To test it, I physically removed the field to get compile errors in parquet-mr where it gets used, and then fixed those errors by removing references to the field. This, however, resulted in TestParquetFileWriter.testMetaDataFile failing. It seems as if summary files still used this field.

Are summary files a real thing or are they deprecated as well? I could hardly find any information on the Internet. I found this [JIRA](https://issues.apache.org/jira/browse/SPARK-15719) about disabling them in Spark, because they were not considered useful, but that's all.